src/engine/world/actor/player/player.ts

```diff
@@ -884,9 +907,9 @@ export class Player extends Actor {
             return;
         }
 
-        const missingRequirements = this.missingItemEquipRequirements(itemDetails);
-        if (missingRequirements.length) {
-            missingRequirements.forEach( s => {this.sendMessage(s)});
+        const equippable = this.canEquipItem(itemDetails);
+        if (equippable.missingRequirements.length) {
```

Comment:
Probably worth checking against `equipable` now that we have it.
```suggestion
        if (!equippable.equipable) {
```